packages/components/src/RadarChart/RadarChart.stories.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Radar, RadarChart, PolarAngleAxis, customizedTick } from './RadarChart.component';
+
+const ExampleDataSingle = [
+    { axis: 'Validity', A: 4 },
+    { axis: 'Social curation', A: 3 },
+    { axis: 'Completeness', A: 2 },
+    { axis: 'Discoverability', A: 1 },
+    { axis: 'other', A: 1 }
+];
+
+const ExampleDataMultiple = [
+    { axis: 'Validity', A: 4, B: 2 },
+    { axis: 'Social curation', A: 3, B: 5 },
+    { axis: 'Completeness', A: 2, B: 1 },
+    { axis: 'Discoverability', A: 1, B: 3 },
+    { axis: 'other', A: 1, B: 3 }
+];
+
+const ExampleDataClickable = [
+    { axis: 'Validity', A: 4 },
+    { axis: 'Social curation', A: 3 },
+    { axis: 'Completeness', A: 2 },
+    { axis: 'Discoverability', A: 1 },
+];
+
+const chartDomain = [0, 5];
+
+/**
+ * A click function provided by the host app
+ */
+const onClick = function () {
+  document.getElementById('clickMessage').innerHTML = "Good job! You clicked " + event.target.innerHTML + ".";
+};
+
+const stories = storiesOf('Data/Dataviz/RadarChart', module);
+stories
+	.addDecorator(story => <div className="col-lg-12">{story()}</div>)
+	.add('Radar Chart Single', () => (
+		<div>
+			<h2>Single Object</h2>
+      <p>A radar chart showing axis scores for a single object</p>
+			<RadarChart data={ExampleDataSingle} domain={chartDomain}>
+        <PolarAngleAxis dataKey="axis" />
```

Comment:
You can consider supporting that in the component (if I understand the idea well, You already add PolarGrid and PolarRadiusAxis directly).

Reply:
In the latest code we expose PolarAngleAxis so that we can pass in a custom function to render the labels/ticks.